kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/GetOrderPollingCall.java

```diff
@@ -38,10 +41,14 @@ private void getOrder(int pollingIndex) {
                     callback.onResponse(order);
                 }
             } else {
-                callback.onFailure(new TimeoutException());
+                callback.onFailure(ErrorUtil.getTimeoutException());
```

Comment:
getTimeoutException will return API exception but this is not necessarily true - there could be other problem if the order is still pending